src/main/resources/db/mysql/petclinic_db_setup_mysql.txt

```diff
@@ -23,7 +23,15 @@
 
 3) Run the app with `spring.profiles.active=mysql` (e.g. as a System property via the command
    line, but any way that sets that property in a Spring Boot app should work).
+   
+4) For newer versions of Spring Boot, those will fork the JVM that executes the webserver,
+   which means properties on the command line are lost. You can use:
+    
+   mvn spring-boot:run\
+   -Dspring-boot.run.jvmArguments="-Dspring.profiles.active=mysql -Dspring.datasource.initialization-mode=always"
```

Comment:
There is a dedicated property to set a profile, `- Dspring-boot.run.profiles=mysql`. I don't think we should mention datasource initialization again (as it should only be invoked the first time you run the app).

Reply:
good point.